packages/frontend-2/assets/css/fonts.css

```diff
@@ -0,0 +1,74 @@
+/* cyrillic-ext */
```

Comment:
@fabis94 I can include this as well in dui3 so it has the correct font (if not it will fallback on the fonts we are already using it), but I dont know how to test that. Should I change it or leave it as is for now?